magenta-lib/src/main/scala/magenta/deployment_type/DeploymentType.scala

```diff
@@ -17,27 +17,27 @@ trait DeploymentType {
   val paramsList = mutable.Map.empty[String, Param[_]]
   lazy val params = paramsList.values.toSeq
   def perAppActions: PartialFunction[String, DeploymentPackage =>
-    (Lookup, DeployParameters, Stack) => List[Task]]
+    (DeployReporter, Lookup, DeployParameters, Stack) => List[Task]]
```

Comment:
My vague ordering here is parameters that are more riff-raff related to the left and more specific deploy related to the right. Platform => Specific Deploy. I'm not wedded to it though.